main.go

```diff
@@ -67,8 +68,46 @@ func usage() {
 }
 
 func main() {
+	log.SetFlags(0)
+	log.SetPrefix("")
+
 	flag.Usage = usage
 	flag.Parse()
+
+	// do some basic validation on the outfile format
+	// indexing is safe because we're searching for an ascii char
```

Comment:
Yes I considered using templates here but opted for what I considered the simpler approach, namely that the format must contain exactly one occurrence of the well-known `%v` verb. Turns out I forgot to include mention of that in the flag description string. So the validation is complete in as much as it satisfies the constraints I had in mind... but then I didn't document those 👎 

The terribly unscientific reason for requiring exactly one occurrence was: 

* zero occurrences will lead to clashes (similar argument to the [renaming rules](https://github.com/ncw/gotemplate#renaming-rules))
* more than one occurrence would be strange

Does this constraint make sense from your perspective?